crates/gen/src/class32.rs

```diff
@@ -0,0 +1,21 @@
+use crate::*;
+use squote::{quote, TokenStream};
+
+#[derive(Debug)]
+pub struct Class32 {
+    pub name: TypeName,
+}
+
+impl Class32 {
+    pub fn from_type_name(name: TypeName) -> Self {
+        Self { name }
+    }
+
+    pub fn gen(&self) -> TokenStream {
+        quote! {}
+    }
+
+    pub fn dependencies(&self) -> Vec<winmd::TypeDef> {
+        Vec::new()
```

Comment:
Is this just for consistency or will these eventually return something meaningful?

Reply:
Yes, just placeholder as all the types need a `dependencies` function and will soon return its dependencies.